src/django_idom/components.py

```diff
@@ -68,56 +68,83 @@ async def async_render():
                         "src": reverse("idom:view_to_component", args=[dotted_path]),
                         "loading": "lazy",
                     }
-                )
+                )  # type: ignore
             )
             return
 
         # Render Check 2: Async function view
         elif iscoroutinefunction(view):
-            view_html = await view(request_obj, *args, **kwargs)
+            view_html = await view(request_obj, *_args, **_kwargs)  # type: ignore
 
         # Render Check 3: Async class view
         elif getattr(view, "view_is_async", False):
-            view_or_template_view = await view.as_view()(request_obj, *args, **kwargs)
+            view_or_template_view = await view.as_view()(request_obj, *_args, **_kwargs)  # type: ignore
             if getattr(view_or_template_view, "render", None):  # TemplateView
                 view_html = await view_or_template_view.render()
             else:  # View
                 view_html = view_or_template_view
 
         # Render Check 4: Sync class view
         elif getattr(view, "as_view", None):
-            async_cbv = database_sync_to_async(view.as_view())
-            view_or_template_view = await async_cbv(request_obj, *args, **kwargs)
+            async_cbv = database_sync_to_async(view.as_view())  # type: ignore
+            view_or_template_view = await async_cbv(request_obj, *_args, **_kwargs)
             if getattr(view_or_template_view, "render", None):  # TemplateView
                 view_html = await database_sync_to_async(view_or_template_view.render)()
             else:  # View
                 view_html = view_or_template_view
 
         # Render Check 5: Sync function view
         else:
-            view_html = await database_sync_to_async(view)(request_obj, *args, **kwargs)
+            view_html = await database_sync_to_async(view)(
+                request_obj, *_args, **_kwargs
+            )
 
         # Signal that the view has been rendered
         set_converted_view(
             utils.html_to_vdom(
                 view_html.content.decode("utf-8").strip(),
                 *transforms,
                 strict=strict_parsing,
-            )
+            )  # type: ignore
```

Comment:
Let's `typing.cast` the initial value of converted view to `VdomDict` so we can remove this ignore comment.